utils/rtl-sdr/Makefile

```diff
@@ -6,14 +6,17 @@
 
 include $(TOPDIR)/rules.mk
 
+PKG_SOURCE_PROTO:=git
 PKG_NAME:=rtl-sdr
-PKG_VERSION:=0.6.0
-PKG_RELEASE:=2
+PKG_VERSION:=master
```

Comment:
This gets autogenerated from PKG_SOURCE_DATE and PKG_SOURCE_VERSION. Please remove.